examples/timeseries/timeseries_forecasting_with_lstm_for_temperature_prediction.py

```diff
@@ -0,0 +1,184 @@
+"""
+Title: Timeseries forecasting with LSTM for temperature prediction
+Author: [Falak Shah](https://github.com/falaktheoptimist)
+Date created: 2020/06/28
+Last modified: 2020/06/28
+Description: Predict average next day temperature using LSTM based model.
+"""
+
+"""
+## Introduction
+This script demonstrates how you can use an LSTM model for time series
+prediction. Usage of timeseries_dataset_from_array is explained for
+preparing input data for the model.
+"""
+
+"""
+## Setup
+As of writing this tutorial, timeseries_dataset_from_array is available only
+in TF-nightly.
+"""
+
+# tf_nightly needed for timeseries_dataset_from_array
+#! pip uninstall -y tensorflow
+#! pip install tf-nightly
+
+from matplotlib import pyplot as plt
+from tensorflow import keras
+from tensorflow.keras import layers
+
+import os
+import matplotlib.dates as mdates
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+
+"""
+## Fetch the data
+We will use the [Austin weather data from ](https://www.kaggle.com/grubenm/austin-weather)
+dataset. It provides timeseries of weather related information for Austin.
+
+We will try to predict the Average temperature for next day using temperature
+and date/ month features. The simplicity of this dataset allows us to
+demonstrate the time series use case of LSTM effectively.
+"""
+
+os.system(
```

Comment:
You can use `keras.utils.get_file` for this

Reply:
That's much better. Thanks.